src/components/movie-view/movie-view.jsx

```diff
@@ -0,0 +1,187 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Button, Card, Container, Col } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import axios from "axios";
+
+export class MovieView extends React.Component {
+  //add favorite
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      favoriteMovies: [],
+    };
+
+    this.addFav = this.addFav.bind(this);
+    this.removeFav = this.removeFav.bind(this);
+  }
+
+  removeFav() {
+    const user = localStorage.getItem("user");
+    const token = localStorage.getItem("token");
+    const id = this.props.movie._id;
+    axios
+      .delete(
+        `https://my-flix-api-2022.herokuapp.com/users/${user}/movies/${id}`,
+
+        { headers: { Authorization: `Bearer ${token}` } }
+      )
```

Comment:
The axios delete request needs 3 parameter. It currently sends your header as the request body. instead you can add an empty object as the second parameter.